vm/runtime/src/lib.rs

```diff
@@ -182,9 +183,31 @@ pub trait Syscalls {
         Ok(data_commitment_v1_to_cid(&comm_d))
     }
     /// Verifies a sector seal proof.
-    fn verify_seal(&self, _vi: &SealVerifyInfo) -> Result<(), Box<dyn StdError>> {
-        // TODO
-        todo!()
+    fn verify_seal(&self, vi: &SealVerifyInfo) -> Result<(), Box<dyn StdError>> {
+        let commd = cid_to_data_commitment_v1(&vi.unsealed_cid)?;
+        let commr = cid_to_replica_commitment_v1(&vi.on_chain.sealed_cid)?;
+        let miner_addr = Address::new_id(vi.sector_id.miner);
+        let mut prover_id = ProverId::default();
+        prover_id.copy_from_slice(&miner_addr.payload_bytes());
```

Comment:
```suggestion
        let miner_payload = miner_addr.payload_bytes();
        prover_id[..miner_payload.len()].copy_from_slice(&miner_payload);
```
Is probably what we want then? Probably good to write a test for this function, even just to pass fake values in and assert the verification errors and not anything else.